src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/Resources/ReadResourcesValidation.feature.disabled

```diff
@@ -0,0 +1,56 @@
+# This is a rough draft feature for future use.
+@ignore
+Feature: Resources "Read" Operation validations
+
+        Background:
+            Given the Data Management Service must receive a token issued by "http://localhost"
+              And user is already authorized
+              And request made to "/ed-fi/absenceEventCategoryDescriptors" with
+                  """
+                    {
+                        "codeValue": "Sick Leave",
+                        "description": "Sick Leave",
+                        "effectiveBeginDate": "2024-05-14",
+                        "effectiveEndDate": "2024-05-14",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Sick Leave"
+                    }
+                  """
+             Then it should respond with 201
+
+        @ignore
+        Scenario: Verify existing resources can be retrieved successfully
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors"
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify retrieving a single resource by ID
+        # Replace Endpoint with the required value, this is just an example to make sure Descriptors are working fine
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/16435704274945eeae6c6c45b281aad3"
+             Then it should respond with 200
+
+        @ignore
+        Scenario: Verify response contains expected data
+        # Replace Endpoint with the required value, this is just an example to make sure Descriptors are working fine
+             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/16435704274945eeae6c6c45b281aad3"
+             Then it should respond with 200
+              And the response body is
+                  """
+                    {
+                        "codeValue": "Jury duty",
+                        "description": "Jury duty",
+                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "shortDescription": "Jury duty"
```

Comment:
The response will also have 

```
                        "effectiveBeginDate": "2024-05-14",
                        "effectiveEndDate": "2024-05-14",
```

Reply:
I just checked, and those values are not included in the response body? Am I missing something?

Reply:
These are optional fields. When they are not included in the POST or PUT, or if they are in the message body but with value `null`, then they also won't be included in the response body.

But if you include them in the POST request, then the values will also appear in the GET response.